### ISSUE TITLE
creating new images post filtering?

### ISSUE THREAD

**Issue by emiloski27** [MouseLand/cellpose#359]
Hi,

First off, thank you to the developers for creating such an amazing program. I am using it to generate cell width measurements on a certain population of neurons. We have generated outputs, but want to do post-processing filtering -- for example, we are filtering by Area of neuron. Each neuron has a label from the mask, is there a way to then generate a new image of the neurons that were not filtered out?

Thanks!

**Comment by carsen-stringer** [comment]
Do you mean the masks that cellpose filters by size? You can turn off size filtering by setting `min_size=-1` in the `eval` call. The default filtering is anything below 15 pixels is thrown out. Then you can make two images depending on your cut-off afterwards:

```
from cellpose import models, utils
model = models.Cellpose(model_type='cyto')
channels = [0,0] # specify channels for model
masks = model.eval(img, channels=channels, min_size=-1)[0]

# you can filter by size using our function or something like np.unique(masks, return_counts=True)
size_threshold = 30
masks_big = utils.fill_holes_and_remove_small_masks(masks, min_size=size_threshold)
masks_small = masks[np.logical_and(masks>0, masks_big==0)]
```

going to close this issue let me know if you still have problems
